src/content/docs/en/guides/integrations-guide/cloudflare.mdx

```diff
@@ -248,6 +248,62 @@ export default defineConfig({
 });
 ```
 
+### `workerEntryPoint`
+<p>
+**Type:** `{path: string | URL, exports: string[]}`
+**Default:** `{ path: '@astrojs/cloudflare/entrypoints/server.js', exports: ['default'] }`
+<Since v="12.6.0" />
```

Comment:
This is the adapter version, not core version. For now we used `<Since>` only for core versions. Not sure if the users might be confused now.

Reply:
Great observation! We can actually specify a particular integration for the version! (See the [`<Since />` documentation](https://contribute.docs.astro.build/reference/custom-components/#since-component).

```suggestion
<Since v="12.6.0" pkg="@astrojs/cloudflare"/>
```